### PR TITLE
release: v3.2.0a1

### LINKED ISSUE #3939: global: invenio v3.2 alpha release with files bundle
Make an **alpha release** of Invenio v3.2 which includes the new Files bundle:

Invenio-Files-REST v1.0.0
Invenio-Records-Files v1.0.0
Invenio-Previewer v1.0.0
Invenio-IIIF v1.0.0

### PR DESCRIPTION
**Pull request by pazembrz**
Packages invenio-previewer and invenio-iiif are still in alpha version.
Remove tests for ES2 as it is deprecated.

closes #3939

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,8 +32,6 @@ env:
     - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.2.tar.gz"
     - ES_HOST=127.0.0.1
   matrix:
-    # ES2 + SQLite
-    - REQUIREMENTS=lowest EXTRAS=all,sqlite,elasticsearch2 SQLALCHEMY_DATABASE_URI="sqlite:///test.db" ES_URL=$ES2_DOWNLOAD_URL
```
Comment (kprzerwa) [inline .travis.yml]: I think you an also remove the `ES2_DOWNLOAD_URL` from above

Comment (pazembrz) [reply .travis.yml]: :+1:

**Comment by roksys** [comment]
Could you modify the commit message? 

Some good examples:

- https://github.com/inveniosoftware/invenio/commit/d2c1f1221671b6ff645ae9053c67f115d45e7e8d
- https://github.com/inveniosoftware/invenio/commit/52a76388522f9392d1a45e40e2dd5d5531003ccd

**Comment by pazembrz** [comment]
Commit message updated.

**Pull request merged**